resources/kiali/values.yaml

```
@@ -298,7 +306,8 @@ kiali:
     # a single, specific Kiali version, thus this setting may have no effect depending on how the
     # operator itself was configured.
     #    ---
-      image_name: "eu.gcr.io/kyma-project/tpi/kiali"
+    #.   ATTENTION: this setting is will not be effective as it got replaced by the global.images.kiali setting. 
```

```suggestion
    #.   ATTENTION: This setting will not be effective. It has been replaced with the global.images.kiali setting. 
```

Are you sure this is the correct line for the Attention note? Why not put it right after line 300 so users can stop reading if they aren't curious?